utils/test_utils.ts

```diff
@@ -220,8 +220,23 @@ export async function cleanupDatabase(
   datastore: Datastore,
 ): Promise<void> {
   await Promise.all([
-    db._builds.deleteMany({}),
     db._modules.deleteMany({}),
+    (async () => {
+      const query = await datastore.db.query(
+        datastore.db.createQuery(kinds.LEGACY_BUILDS),
+      );
+      const mutations = query.map((entry) => ({
+        delete: objectGetKey(entry)!,
+      }));
+
+      for await (
+        const _ of datastore.db.commit(mutations, {
+          transactional: false,
+        })
+      ) {
+        //
+      }
+    })(),
```

Comment:
Shouldn't be wrapped in `try/catch`?

Reply:
its just for testing; if this fails for some reason, then something would be wrong with either the db or tests, so throwing is ok